### ISSUE TITLE
library.json has wrong platform

### ISSUE THREAD

**Issue by fmuntean** [philbowles/STM32AsyncTCP#1]
the library platforms tag still contains the "espressif8266" I think should be "stm32"

**Comment by jcw** [comment]
Yep, I bumped into this as well. _This quick-and-easy fix would be nice ..._

UPDATE: the other change I had to make, was to remove `"build": { "libcompatmode": 2 }`. Then it works.

**Comment by mostafahk** [comment]
Please gave me permission to fix it or do it your self:

replace library.json:

{
  "name":"STM32AsyncTCP",
  "description":"Asynchronous TCP Library for STM32",
  "keywords":"async,tcp",
  "authors":
  {
    "name": "Hristo Gochkov",
    "maintainer": true
  },
  "repository":
  {
    "type": "git",
    "url": "https://github.com/philbowles/STM32AsyncTCP"
  },
  "version": "1.2.2",
  "license": "LGPL-3.0",
  "frameworks": "arduino",
  "platforms": "ststm32"
}
